diff --git a/node_modules/@types/styled-components-react-native/index.d.ts b/node_modules/@types/styled-components-react-native/index.d.ts
index 84b0ad6..759b3ba 100755
--- a/node_modules/@types/styled-components-react-native/index.d.ts
+++ b/node_modules/@types/styled-components-react-native/index.d.ts
@@ -70,8 +70,8 @@ declare module "styled-components/native" {
         ListView: ReactNativeThemedStyledFunction<typeof ReactNative.ListView, T>;
         Modal: ReactNativeThemedStyledFunction<typeof ReactNative.Modal, T>;
         NavigatorIOS: ReactNativeThemedStyledFunction<typeof ReactNative.NavigatorIOS, T>;
-        Picker: ReactNativeThemedStyledFunction<typeof ReactNative.Picker, T>;
-        PickerIOS: ReactNativeThemedStyledFunction<typeof ReactNative.PickerIOS, T>;
+        // Picker: ReactNativeThemedStyledFunction<typeof ReactNative.Picker, T>;
+        // PickerIOS: ReactNativeThemedStyledFunction<typeof ReactNative.PickerIOS, T>;
         Pressable: ReactNativeThemedStyledFunction<typeof ReactNative.Pressable, T>;
         ProgressBarAndroid: ReactNativeThemedStyledFunction<typeof ReactNative.ProgressBarAndroid, T>;
         ProgressViewIOS: ReactNativeThemedStyledFunction<typeof ReactNative.ProgressViewIOS, T>;
